src/Service.Host/client/src/components/PurchaseOrders/CreatePurchaseOrderUt.js

```diff
@@ -198,9 +202,24 @@ function CreatePurchaseOrderUt() {
         id: c.partNumber,
         name: c.partNumber,
         partNumber: c.partNumber,
-        description: c.description
+        description: c.description,
+        stockControlled: c.stockControlled
     }));
 
+    const partSuppliersSearchResults = useSelector(reduxState =>
+        collectionSelectorHelpers
+            .getSearchItems(reduxState.partSuppliers)
+            ?.sort((a, b) => (a.supplierRanking ?? 100) - (b.supplierRanking ?? 100))
+            .map(s => ({
+                id: s.supplierId,
+                displayText: `${s.supplierName} RANK : ${s.supplierRanking}`
```

Comment:
I don't think you need to display supplierRanking here (its often null. Just indicating which one has supplier ranking = 1 (i.e. which is the preferred supplier) is probably better.